lib/pages/new_group/new_group_chat_info.dart

```diff
@@ -212,12 +175,41 @@ class NewGroupChatInfo extends StatelessWidget {
             shape: BoxShape.circle,
           ),
           alignment: Alignment.center,
-          child: Icon(Icons.camera_alt_outlined,
-            color: Theme.of(context).colorScheme.surface,
+          child: ValueListenableBuilder(
+            valueListenable: newGroupController.avatarNotifier,
+            builder: (context, value, child) {
+              if (value == null) {
+                return child!;
+              }
+              return ClipOval(
+                child: SizedBox.fromSize(
+                  size: const Size.fromRadius(28),
+                  child: AssetEntityImage(
+                    value,
```

Comment:
this also load binary to the mem or not?

Reply:
this load into bytes but with a small size of thumbnail image. in this case it is 56x56 px, set by ThumbnailSize option